python/ray/includes/function_descriptor.pxi

```diff
@@ -187,10 +191,13 @@ cdef class PythonFunctionDescriptor(FunctionDescriptor):
             The FunctionDescriptor instance created according to the function.
         """
         module_name = cls._get_module_name(function)
+        args_name = cls._get_args_name(function)
```

Comment:
This is only computed once when the function is registered right?

Reply:
Yeah that's correct.